### DISCUSSION TITLE
Add Connection.

### DISCUSSION THREAD

**Discussion by zraiss** [cuebook/CueObserve#144] [General]
I am under Unix and I tried to create a connection between MYSQL and CueObserve but I still get the connection failed message.
Can you tell me what's the problem?
I even tried to test with SqlServer

**Comment by ankitkpandey** [comment]
Could you please post the logs of the error? Also what exactly do you mean by Unix, do you have a Unix-like OS e.g. Linux or are you using FreeBSD?

**Reply by zraiss** [reply]
I am using ubuntu 20.
And when I make the connection between CueObserve and MSSQL or MYSQL it doesn't work anymore

**Reply by zraiss** [reply]
![Capture d’écran du 2021-09-20 17-39-32](https://user-images.githubusercontent.com/91097421/134141153-5f7c4d00-8c1e-4141-a598-c7dedda48259.png)
![Capture d’écran du 2021-09-20 17-42-11](https://user-images.githubusercontent.com/91097421/134141162-44ce7fdf-219d-4ea4-9025-7ea62888ad7d.png)
![Capture d’écran du 2021-09-20 22-01-29](https://user-images.githubusercontent.com/91097421/134141166-cfad255d-4456-4963-8b16-ce112f833604.png)

**Reply by zraiss** [reply]
on the last image, the error displayed

**Reply by vincue** [reply]
Did you pass the flag `--network="host"` for giving container access to local system services?

**Reply by zraiss** [reply]
No, how can I do'It?

**Reply by vincue** [reply]
Pass it with docker command like `docker run --network="host" -p 3000:3000 cuebook/cueobserve`.

**Reply by zraiss** [reply]
On "host" I write the address of the host? which in my case is "localhost"?

**Reply by vincue** [reply]
Just paste as is `--network="host"`

**Comment by zraiss** [comment]
I typed the command that you shared with me, this is the output that I got
![Capture d’écran du 2021-09-22 15-29-38](https://user-images.githubusercontent.com/91097421/134352920-5cc85226-7b21-4229-bd06-43795a30f9aa.png)

**Reply by vincue** [reply]
There must be something already running on your `localhost:3000`?

**Reply by zraiss** [reply]
Yes I have runner CueObserve image using docker with port 3000

**Reply by zraiss** [reply]
I closed my laptop and I restarted it after I typed the command that you shared with me, but still the same problem.
This is what I got as output on Terminal 
![Capture d’écran du 2021-09-25 21-35-28](https://user-images.githubusercontent.com/91097421/134784106-b69b325e-bc4e-4918-989a-3d4cdee4d057.png)

**Reply by vincue** [reply]
In CueObserve > Connection Drawer, use Host `127.0.0.1`, there seem to some problem with ~~`localhost`~~
